### ISSUE TITLE
Balance Sheet and Income report negative values with GIFIs

### ISSUE THREAD

**Issue by ylavoie** [ledgersmb/LedgerSMB#3296]
Using GIFI numbers instead of standard shows various amounts as negative instead of their proper value.
The report was selected by period for any given year, and with or without previous years shown.
The same reports show correctly on 1.4.

**Comment by ehuelsmann** [comment]
Do the GIFI amounts showing the wrong sign add up only income or also expense accounts?

**Comment by ylavoie** [comment]
Demo is showing individual amounts as negative but the total as positive for assets and expenses. I asked for a report from 01-01-2007, by GIFIs, with account hierarchies. See:
```
Balance Sheet
RP_LegalResidence_TaxMapParcel6C-022
8 Brookside Dr. Merrimack NH 03054-3036
01-01-2017
Assets	
-1,688,137.79
1-01-200 - Banco General	-10,999,876.55
111 -	235,326.50
Assets	12,452,687.84
Liabilities	
202,651.95
Liabilities	202,651.95
Equity	
11,254,484.66
Equity	11,254,484.66
Expenses	
-154,367.43
E - Expenses	154,367.43
Income	
1,148,896.74
I - Income	1,148,896.74
```

**Comment by ehuelsmann** [comment]
Fixed in 3430c58
